ginga/util/loader.py

```diff
@@ -118,5 +118,6 @@ def add_loader(mimetype, loader):
     add_loader(mimetype, load_asdf)
 
 # ### RGB ###
-for mimetype in ['image/jpeg', 'image/png', 'image/tiff', 'image/gif']:
+for mimetype in ['image/jpeg', 'image/png', 'image/tiff', 'image/gif',
+                 'image/ppm', 'image/pnm', 'image/pbm', 'image/pnm']:
```

Comment:
`pnm` was listed twice. Did you mean something else?
```suggestion
                 'image/ppm', 'image/pnm', 'image/pbm']:
```

Reply:
I think it's fine to include a mention of STScI there. Thanks!

Reply:
Good catch, @pllim.  I think that is in there in error.  I'll update the PR.